includes/class-freemius.php

```diff
@@ -3610,6 +3608,100 @@ private static function _load_required_static() {
         #region Clone
         #--------------------------------------------------------------------------------
 
+        /**
+         * Tries to recover the install of a newly created subsite or resolve it if it's a clone.
+         * 
+         * @author Leo Fajardo (@leorw)
+         * @since 2.5.0
+         */
+        private function maybe_resolve_new_subsite_install_automatically() {

```

Comment:
@fajardoleo can we move this method, and the other newly created helper methods into the `FS_Clone_Manager`? That way we'll also be able to revert the visibility of the methods in `FS_Clone_Manager` that are used back to `private`.